src/cmd/helm.rs

```diff
@@ -58,7 +59,7 @@ pub enum HelmError {
 
 #[derive(Debug)]
 pub struct Helm {
-    kubernetes_config: PathBuf,
+    pub kubernetes_config: PathBuf,
```

Comment:
is the pub still necessary ? it seems fishy

Reply:
forgot to remove it after file architecture changed.